app/controllers/articles/favorites_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Articles::FavoritesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_article, only: %i[create destroy]
```

Comment:
```suggestion
  before_action :set_article
```
もはやArticles::XXControllerは必ずarticle_idがあるはずだから必ず実施で良さそう